core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -118,6 +118,9 @@ private RFile() {}
   // Buffer sample data so that many sample data blocks are stored contiguously.
   private static int sampleBufferSize = 10000000;
 
+  // 5 bytes of overhead for fields: value, row, colFamily, colQualifier, colVisibility, and 8 for the timestamp
```

Comment:
would nice to mention what the overhead is... its worst case serialization overhead